src/app/components/dashboard/dashboard-content/dashboard-content.component.ts

```diff
@@ -0,0 +1,104 @@
+import { Component, OnInit } from '@angular/core';
+import {ApiService} from '../../../services/api.service';
+import {AuthService} from '../../../services/auth.service';
+import {GlobalService} from '../../../services/global.service';
+import {ActivatedRoute, Router} from '@angular/router';
+import {EndpointsService} from '../../../services/endpoints.service';
+
+@Component({
+  selector: 'app-dashboard-content',
+  templateUrl: './dashboard-content.component.html',
+  styleUrls: ['./dashboard-content.component.scss']
+})
+export class DashboardContentComponent implements OnInit {
+
+
+  /**
+   * Challenges list
+   */
+  challenges = [];
+
+  /**
+   * Host teams list
+   */
+  hostteams = [];
+
+  /**
+   * Participant teams list
+   */
+  participantteams = [];
+
+  /**
+   * Constructor.
+   * @param endpointsService  EndpointService Injection.
+   * @param route  ActivatedRoute Injection.
+   * @param router  Router Injection.
+   * @param globalService  GlobalService Injection.
+   * @param apiService  ApiService Injection.
+   * @param authService  AuthService Injection.
+   */
+  constructor(private apiService: ApiService,
+              private authService: AuthService,
+              private globalService: GlobalService,
+              private router: Router,
+              private route: ActivatedRoute,
+              private endpointsService: EndpointsService) { }
+
+  /**
+   * Component on initialized.
+   */
+  ngOnInit() {
+    if (!this.authService.isLoggedIn()) {
+      this.router.navigate(['/auth/login']);
```

Comment:
Don't hard-code the URL, `/auth/login`, define a separate variable above or get from elsewhere if available.